content/300-guides/200-deployment/110-deployment-guides/400-deploying-to-aws-lambda.mdx

```diff
@@ -340,6 +340,55 @@ package:
 
 This ensures the packaged archive is as small as possible.
 
+But, if you are using `serverless-webpack`, you cannot use that method.
+
+### Deployment using `serverless-webpack`
+
+If you are using serverless-webpack, you can use the [serverless-webpack-prisma](https://github.com/danieluhm2004/serverless-webpack-prisma) plugin without any additional configuration.
+
+
+1. There is no need to separately copy the schema.prisma file via the `webpack plugin`.
+2. There is no need to add `prisma generate` command to webpack options separately.
+3. Except for the Prisma engine used in Lambda, it is automatically deleted. This can save more than 40mb of capacity.
```

Comment:
Does this mean there is no need for package patterns in `serverless.yml`?

Reply:
It's a little different.

The file created when serverless-webpack runs the webpack script part is not affected by serverless.patterns.

That is why it is necessary to delete them through this library.